cloud/amazon/ec2_elb_lb.py

```diff
@@ -900,6 +921,11 @@ def main():
     if connection_draining_timeout and not elb_man._check_attribute_support('connection_draining'):
         module.fail_json(msg="You must install boto >= 2.28.0 to use the connection_draining_timeout attribute")
 
+    if connection_settings_idletimeout and not self._check_attribute_support('connecting_settings'):
```

Comment:
Should this be `elb_man._check_attribute_support('connection_settings')` instead?